llm/llama-4-finetuning/llama-4-maverick-lora.yaml

```diff
@@ -32,6 +32,9 @@ file_mounts:
   /configs: ./configs
 
 setup: |
+  conda create -n training python=3.10 -y
+  conda activate training
```

Comment:
how about using `uv venv ~/training --seed --python 3.10; source ~/training/bin/activate`, and also use `uv pip install` for the installation below?